src/overlord/Proxy.js

```diff
@@ -307,7 +307,14 @@ export class ProxyOverlord {
         console.log("Proxy finished any pending caching transactions");
     }
 
-    _remoteCall(commandOrString) {
+    // count - the number of collapsed requests
+    allCollapsed(path, count) {
```

Comment:
This code is not specific to collapsed forwarding logic/features, right? Along with the other changes:

```suggestion
    // Wait for the proxy to parse the given number of request headers (containing the given URL path)
    // without satisfying any of those requests.
    // This only works if the received requests cannot be quickly satisfied by the proxy!
    // For example, the parsed requests may be waiting for the server(s) to respond.
    async finishStagingRequests(path, count) {
```